godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/IndexedIterator.kt

```diff
@@ -13,3 +13,20 @@ internal class IndexedIterator<T>(
         return getter(index++)
     }
 }
+
+data class Entry<out K, out V>(val key: K, val value: V)
```

Comment:
why not use `Pair`?

Reply:
first used Pair. But the properties to access the two elements are .first and .second.
Entry does use .key and .value